public/app/plugins/datasource/azuremonitor/components/ConfigEditor/AzureCredentialsForm.tsx

```diff
@@ -3,30 +3,50 @@ import React, { useMemo } from 'react';
 import { SelectableValue } from '@grafana/data';
 import { ConfigSection } from '@grafana/experimental';
 import { Select, Field } from '@grafana/ui';
+import { getAzureClouds } from '@grafana/azure-sdk';
 
 import { selectors } from '../../e2e/selectors';
 import { AzureAuthType, AzureCredentials } from '../../types';
 
 import { AppRegistrationCredentials } from './AppRegistrationCredentials';
 import CurrentUserFallbackCredentials from './CurrentUserFallbackCredentials';
 
+const legacyAzureClouds: SelectableValue[] = [
+  { value: 'azuremonitor', label: 'Azure' },
+  { value: 'govazuremonitor', label: 'Azure US Government' },
+  { value: 'chinaazuremonitor', label: 'Azure China' },
+];
+
 export interface Props {
   managedIdentityEnabled: boolean;
   workloadIdentityEnabled: boolean;
   userIdentityEnabled: boolean;
   credentials: AzureCredentials;
   azureCloudOptions?: SelectableValue[];
-  legacyAzureCloudOptions?: SelectableValue[];
   onCredentialsChange: (updatedCredentials: AzureCredentials) => void;
   disabled?: boolean;
   children?: JSX.Element;
 }
 
+export function getAzureCloudOptions(): Array<SelectableValue<string>> {
```

Comment:
@Jeralan this won't work as expected from a quick test on my side. I think we should go with @JonCole's suggestion and normalise the legacy clouds on the frontend instead, wdyt?